src/test/java/pl/touk/sputnik/CliOptionsTest.java

```diff
@@ -1,22 +1,23 @@
 package pl.touk.sputnik;
 
 import org.apache.commons.cli.CommandLine;
-import org.junit.Test;
-import org.junit.Assert;
+import org.junit.jupiter.api.Test;
 import pl.touk.sputnik.configuration.CliOption;
 import pl.touk.sputnik.configuration.CliWrapper;
 
-public class CliOptionsTest {
+import static pl.touk.sputnik.SputnikAssertions.assertThat;
 
-    private CliWrapper fixture = new CliWrapper();
+class CliOptionsTest {
+
+    private final static String SAMPLE_CONFIG = "/home/spoonman/sputnik/conf.properties";
```

Comment:
\[Checkstyle\] ERROR: 'static' modifier out of order with the JLS suggestions\.

Reply:
this suggestion is worth applying

Reply:
Done